specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentralapps.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "1.0.0",
+    "version": "preview",
```

Comment:
Who has signed off on this? Has this been already released with the incorrect version?

Reply:
The initial docs that were published had the wrong version. The APIs are, and have always been in preview